### PR TITLE
border when creating arrows

### PR DESCRIPTION
**Pull request by kentis**

### CODE REVIEW

File: src/components/ArrowIndicator/ArrowIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-nested-ternary */
```
Comment (boyum) [inline src/components/ArrowIndicator/ArrowIndicator.tsx:1]:
```suggestion
```

File: src/components/ArrowIndicator/ArrowIndicatorContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-nested-ternary */
```
Comment (boyum) [inline src/components/ArrowIndicator/ArrowIndicatorContainer.tsx:1]:
```suggestion
```

File: src/components/ArrowIndicator/ArrowIndicator.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import * as React from "react";
+import { Position } from "../../types/Position";
+
+export type ArrowIndicatorProps = {
+  position: Position;
+  cellSize: number;
+  gapSize: number;
+};
+
+// TODO: Share code with h5p-topic-map instead of duplicating
```
Comment (boyum) [inline src/components/ArrowIndicator/ArrowIndicator.tsx:11]: This isn't used in the client code as well, is it?

```suggestion
```

File: src/components/ArrowIndicator/ArrowIndicator.module.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.arrowSvg {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  z-index: 10;
+  pointer-events: none;
+}
```
Comment (boyum) [inline src/components/ArrowIndicator/ArrowIndicator.module.scss:7]: Is this file necessary?

```suggestion
```

Comment (kentis) [reply src/components/ArrowIndicator/ArrowIndicator.module.scss:7]: I think so.. or at least the stylings it contains...

File: src/components/Grid/Grid.tsx [hunk 5]
```diff
@@ -853,6 +857,20 @@ export const Grid: FC<GridProps> = ({
     [arrowItems, renderArrow],
   );
 
+  const childrenArrowIndicators = useMemo(
+    () =>
+      arrowIndicators.map(position => {
+        return (
+          <ArrowIndicator
+            position={position}
```
Comment (boyum) [inline src/components/Grid/Grid.tsx:865]: A `key` property with a unique value should be added to all components made in a loop.

```suggestion
            key={`${position.x}-${position.y}`}
            position={position}
```

**Review by boyum — APPROVED** [review]
